gizmo.py

```diff
@@ -0,0 +1,81 @@
+#Ex 2
+def hello(name,country='Finland'):
+    print('Hello ',name,', how are things in ',country,'?',sep='')
+        
+#Ex 3
+def spell(word):
+    for letter in range(0,len(word)-1):
+        print(word[letter],end='.')
+    print(word[len(word)-1])
+
+#Ex 4
+def relative_path(identifiers):
+    names=[ ]
+    for element in identifiers:
+        names.append(f"./subjects/mock_recording_{element}.rec")
+    return names
+
+#Ex 5, 6 and 7
+class Gizmo:
+    def __init__(self, name):
+        self.name = name 
+        
+    def speak(self):
+        print(self.name)
+        
+#Ex 8, 9 and 10
+def multiplication_table(zero_out_multiples=None):    
+    """
+    Returns the multiplication tables from 1 to 12 in an array.
+
+    This function returns a 12*12 array containing the numbers corresponding 
+    to the multiplication tables from 1 to 12. It is possible to ask the 
+    tables setting to 0 all multiples of any integer in the array.
+
+    Parameters
+    ----------
+    zero_out_multiples : int or None, optional
+        Non-zero value multiples will be set to 0, or None by default.
+    
+    Returns
+    -------
+    mat : array
+        Contains the multiplication tables from 1 to 12 with the selected 
+        values set to 0
+
+    """
+    
+    import numpy as np
+    mat=np.outer(np.arange(1,13,1),np.arange(1,13,1))
+    if ( zero_out_multiples!= None):  
```

Comment:
```suggestion
    if zero_out_multiples is not None:
```

Technically, you should use the `is` keyword to compare against `None`. `is` compared object identity, whereas `==` compares values. Since `None` is a singleton object, all `None` objects are the same object.